README.md

```diff
@@ -290,6 +291,7 @@ Options for the `Builder` class
     * `xmldec.standalone` standalone document declaration: true or false
   * `doctype` (default `null`): optional DTD. Eg. `{'ext': 'hello.dtd'}`
   * `headless` (default: `false`): omit the XML header. Added in 0.4.3.
+  * `cdata` (default: `false`): wrap text nodes in `<![CDATA[ ... ]]>` 
```

Comment:
> I guess we could try adding a normal text node, then if it fails (throws an error), we can fallback to trying with a CDATA wrap?

This should never happen, since the only things that *have* to be escaped are &, < and >. So it only really makes sense if the text node contains any of these. I guess checking the contents for these and only then soing CDATA if requested would work best.

> Do you want me to bump the version as part of this PR?

No, just say, added in 0.4.x, for whatever x is when merged. So people know they don't have to try this option in 0.4.1 or something. This is only for people to know when they can depend on this feature and when not.

(Still, please do not take this as a discouragement - it is a really good PR and I want to really nail in when merging, so users of xml2js get the best possible compromise. So thanks in advance!)